integration/nwo/components.go

```diff
@@ -69,3 +71,12 @@ var RequiredImages = []string{
 	runner.KafkaDefaultImage,
 	runner.ZooKeeperDefaultImage,
 }
+
+func ModuleCCPath() string {
+	ccPath := filepath.Join(os.Getenv("PROJECT_ROOT_DIR"), "integration/chaincode/module/main.go")
+	curExPath, err := filepath.Abs(".")
+	Expect(err).NotTo(HaveOccurred())
+	ccPath, err = filepath.Rel(curExPath, ccPath)
+	Expect(err).NotTo(HaveOccurred())
+	return ccPath
+}
```

Comment:
See the commit message. This is kind of a hack, but I see it as playing by the recommended rules.

Reply:
It's a complete hack. :) What rules are you referring to?

There are a couple of problems here but the one that bothers me most is the introduction of an environment variable will impact anyone trying to benefit from the "clone and go" model of using modules.

Reply:
Rule of giving build a relative reference will treat it as a thing to build and not a package.

I need to look closer into the proposal to invoke `go env GOMOD` https://github.com/sykesm/fabric/commit/a96bd5c19843774cc4ea5c470fb8a3d284846b5b#diff-a1e8309c32c99d205611d0d8d4187418R68
Very much potential.